src/index.js

```diff
@@ -84,15 +84,17 @@ export default class ImageTransformer extends React.Component {
                 : undefined,
             keyAccumulator: 1,
             source: undefined,
-            enableOrientation: false
+            enableOrientation: false,
+	    _dimensionsChangeListener: null,
```

Comment:
IMO, it seems to be better if `_dimensionsChangeListener` be a class property, not a state.